docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,12 +1,70 @@
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
 The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
 
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install [Elastic for Security Operations (SecOps)] https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f from the ServiceNow Store.
+* Create a ServiceNow integration user and assign it the appropriate roles.
+* Create a CORS rule.
+
+*Create a ServiceNow integration user*
+
+To ensure authenticated communication between Elastic and ServiceNow, create a ServiceNow integration user and assign it the appropriate roles. 
+
+. Go to *System Security -> Users*.
+. Click *New*.
+. Complete the form.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `import_transformer`
+* `personalize_choices`
+* `sn_si.basic`
+* `x_elas2_sir_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and ServiceNow. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+. Configure the rule settings to use the Elastic SIR API and the Kibana URL:
+* *REST API*: Choose `Elastic SIR API [x_elas2_sir_int/elastic_api]`.
+* *Domain*: Enter the Kibana URL.
+. Go to the *HTTP methods* tab and select *GET*. This chooses the GET HTTP method.
+. Click *Submit* to create the rule.
+
+[float]
+[[servicenow-sir-connector-update]]
+==== Update a deprecated ServiceNow SecOps connector
+
+ServiceNow SecOps connectors created in 7.15 version are marked as deprecated after you upgrade to 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when you select them.
+
+[role="screenshot"]
+image::management/connectors/images/servicenow-sir-update-connector.png[Shows deprecated ServiceNow connectors]
+
+IMPORTANT: Deprecated connectors will continue to function with the rules they were added to and can be assigned to a new rule. However, we strongly recommend updating the deprecated connector or <<creating-new-connector, creating a new one>> to ensure you have access to connector enhancements, such as updating incidents.
+
+To update a deprecated connector:
+
+. Open the main menu and go to *Stack Management -> Rules and connectors -> Connectors*.
+. Select the deprecated connector to open the connector Edit flyout.
+. In the warning message, click *Update this connector*.
+. Complete the guided steps in the Edit connector flyout.
```

Comment:
based on lines 60 and 62 I'm not sure if we're calling it the "connector Edit flyout", or the "Edit connector flyout"

Reply:
Originally, "connector Edit flyout" was used, but I've since changed it to "Edit connector flyout". I think this might still be up for discussion though (see https://github.com/elastic/security-docs/pull/1188#discussion_r745927802).